lbz/handler_base.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from typing import Generic, TypeVar
+
+from lbz.misc import deprecated
+
+T = TypeVar("T")
+
+
+class HandlerBase(Generic[T]):
```

Comment:
Also, this should be `BaseHandler` to make it more sense from the English language perspective 😉